src/components/button-group/__tests__/button-group.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render } from '@testing-library/react'
+import { ButtonGroup } from '../button-group'
+import { Button } from '@/components/button/button'
+
```

Comment:
**todo:** Snapshotting the result of the `render` function means we capture React Testing Library's API. This problem is described in #157. Can you please update these tests to use the `asFragment` helper returned by `render`?

Reply:
I agree with this and will update it.